diagrams/create_request_uri.md

```diff
@@ -0,0 +1,50 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
```

Comment:
There are a few parameters that don't map to the text like `v_nonce`, `w_epk_key`. This diagram also assumes a certain architecture choices that are not introduced by the generic diagram in the main text. Also, the diagram uses `response_code` which again is not normative. 

Also the terminology is not clear. Specifically, signed authorization request, signed request object, presentation request and presentation response is inconsistent with other specifications like JAR and OAuth2.